website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -187,6 +189,11 @@ variable "orca_default_bucket" {
   description = "Default ORCA S3 Glacier bucket to use."
 }
 
+variable "orca_restore_retrieval_type" {
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
moved